components/dropdown/dropdown-popover-mixin.js

```diff
@@ -113,7 +113,7 @@ export const DropdownPopoverMixin = superclass => class extends LocalizeCoreElem
 	static get styles() {
 		return [super.styles, css`
 			.dropdown-content-layout {
-				align-items: flex-start;
+				align-items: stretch;
```

Comment:
This addresses issue where contents of `d2l-filter` do not fill available space.